sdk/loadtestservice/Azure.ResourceManager.LoadTestService/src/autorest.md

```diff
@@ -0,0 +1,62 @@
+# Generated code configuration
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+``` yaml
+
+azure-arm: true
+csharp: true
+library-name: LoadTestService
+namespace: Azure.ResourceManager.LoadTestService
+require: https://github.com/Azure/azure-rest-api-specs/blob/ec278eb936001b993e0413d66d8cc88e73540331/specification/loadtestservice/resource-manager/readme.md
+tag: package-2022-12-01
```

Comment:
What's the service ga day?

Reply:
@ArthurMa1978 We're targeting to go to GA at max before the end of the year.